src/test/suite/extension.test.ts

```diff
@@ -0,0 +1,15 @@
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+// import * as myExtension from '../../extension';
+
+suite('Extension Test Suite', () => {
+	vscode.window.showInformationMessage('Start all tests.');
+
+	test('Sample test', () => {
+		assert.strictEqual(-1, [1, 2, 3].indexOf(5));
+		assert.strictEqual(-1, [1, 2, 3].indexOf(0));
+	});
+});
```

Comment:
Is the idea that we'd use this test suite to test the hovering functionality? I'm not super familiar with the API VS Code exposes. Does it let us trigger hover states for words in a file to be able to test something like that?

Reply:
Yep, this is my plan. Though I have't yet done enough research to determine if hovering related API is exposed, but I would guess VScode allows user to select a word in the testing API.

Reply:
Thanks for doing the testing infrastructure set up, the initiative is great! I'm in favor of leaving this as is for now and coming back to it later (only if you want to go through with the plan, I don't have very strong feelings about it either way)